src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1486,16 +1488,22 @@ static  inline fsal_status_t kvsfs_getattrs(struct fsal_obj_handle *obj_hdl,
 
 	myself =
 		container_of(obj_hdl, struct kvsfs_fsal_obj_handle, obj_handle);
-
-	retval = cfs_getattr(myself->cfs_fs, &cred,
```

Comment:
As per my comment on another PR, cfs_getattr is required function. So keep this call with required modification in parameters.

Reply:
Done